examples/compute_head_direction.py

```diff
@@ -0,0 +1,397 @@
+# We will do this for a small time window to make the plot more readable.
+
+# Time window to restrict the plot
+time_window = slice(54.9, 55.1)  # seconds
+
+fig, ax = plt.subplots()
+
+# Plot the computed head-to-snout vector originating from the ears midpoint
+ax.quiver(
+    midpoint_ears.sel(space="x", time=time_window),
+    midpoint_ears.sel(space="y", time=time_window),
+    head_to_snout.sel(space="x", time=time_window),
+    head_to_snout.sel(space="y", time=time_window),
+    midpoint_ears.sel(time=time_window).time.values,  # color by time
+    angles="xy",
+    scale=1,
+    scale_units="xy",
+    headwidth=4,
+    headlength=5,
+    headaxislength=5,
+    label="Head-to-snout vector",
+)
+
+# Plot midpoint between the ears within the time window
+plot_trajectory(
+    midpoint_ears.sel(time=time_window),
+    ax=ax,
+    s=60,
+    label="ears midpoint",
+)
+
+# Plot the snout position within the time window
+plot_trajectory(
+    snout.sel(time=time_window),
+    ax=ax,
+    s=60,
+    label="snout",
+    marker="*",
+    c="r",
+)
+
+ax.set_title("Zoomed in head-to-snout vectors")
+ax.invert_yaxis()  # invert y-axis to match image coordinates
+ax.legend(loc="upper left")
+
+
+# %%
+# 2D vectors in polar coordinates
+# -------------------------------
+# Now that we have the head-to-snout vector, we can compute its orientation
+# angle in 2D space. A convenient way to achieve that is to convert the
+# vector from cartesian to polar coordinates using the
+# :func:`cart2pol()<movement.utils.vector.cart2pol>` function.
+
+from movement.utils.vector import cart2pol, pol2cart
+
+head_to_snout_polar = cart2pol(head_to_snout)
+
+print(head_to_snout_polar)
+
+# %%
+# Notice how the resulting array has a ``space_pol`` dimension with two
+# coordinates: ``rho`` and ``phi``. These are the polar coordinates of the
+# head vector.
+#
+# The coordinate ``rho`` is the norm (i.e., magnitude, length) of the vector.
+# In our case, the distance from the midpoint between the ears to the snout.
+# The coordinate ``phi`` is the orientation of the head vector relative to the
+# positive x-axis, and ranges from :math:`-\pi` to :math:`\pi` in radians.
+# (following the `atan2 <https://en.wikipedia.org/wiki/Atan2>`_ convention).
+#
+# In our coordinate system, ``phi`` will be
+# positive if the shortest path from the positive x-axis to the vector is
+# clockwise. Conversely, ``phi`` will be negative if the shortest path from
+# the positive x-axis to the vector is anti-clockwise.
+#
+# .. image:: ../_static/Cartesian-vs-Polar.png
+#   :width: 600
+#   :alt: Schematic comparing cartesian and polar coordinates
+
+# %%
+# ``movement`` also provides a ``pol2cart`` utility to transform
+# data in polar coordinates back to cartesian.
+# Note that the resulting ``head_to_snout_cart`` array has a ``space``
+# dimension with two coordinates: ``x`` and ``y``.
+
+head_to_snout_cart = pol2cart(head_to_snout_polar)
+print(head_to_snout_cart)
```

Comment:
Maybe instead of printing (which potentially doesn't add much) we can assert that the output is the same as `head_to_snout`.

Or if we want to keep the print statement, we can add a line in text to notice that the space coordinates are back to 'x' and 'y'.